src/sql/parts/notebook/notebookActions.ts

```diff
@@ -229,33 +232,74 @@ export class AttachToDropdown extends SelectBox {
 	public updateModel(model: INotebookModel): void {
 		this.model = model;
 		model.contextsChanged(() => {
-			if (this.model.clientSession.kernel && this.model.clientSession.kernel.name) {
-				let nameLower = this.model.clientSession.kernel.name.toLowerCase();
-				let currentKernelSpec = this.model.specs.kernels.find(kernel => kernel.name && kernel.name.toLowerCase() === nameLower);
-				this.loadAttachToDropdown(this.model, currentKernelSpec.display_name);
+			let kernelDisplayName: string = this.getKernelDisplayName();
+			if (kernelDisplayName) {
+				this.loadAttachToDropdown(this.model, kernelDisplayName);
 			}
 		});
 	}
 
+	private updateAttachtoDropdown(model: INotebookModel): void {
```

Comment:
Need to capitalize the "to" in the middle.

Reply:
I was considering 'Attachto' as a single word from a usability standpoint in Notebooks.
@kevcunnane @chlafreniere - What do you think?

Reply:
Yeah it's 2 words. Just like in the UI it's `Attach To:` so it should be camel-cased everywhere in the code

Reply:
Spent about 2 minutes to think while coding it yesterday. Should have gone straight :)